samples/less/samples.less

```diff
@@ -2,7 +2,7 @@
 @components-dir: "../../node_modules/cksource-samples-framework/components";
 
 // Good 'ol Lesshat.
-@import "../../node_modules/cksource-samples-framework/node_modules/lesshat/build/lesshat";
+@import "../../node_modules/lesshat/build/lesshat";
```

Comment:
If we were to use it that way we need to add `lesshat` as a dev dependency to our project too.

Reply:
I'm pretty sure that I get error multiple times because this file wasn't present. It seems that I had some repository problem after reinstallation it's there.

Reply:
It depends on npm resolution. Currently node will put deps of your deps in your `node_modules` directory, but will that be the case in a year from here? We don't know.

Thus it needs to be added to deps.

Reply:
I wrongly interpret the previous comment then ;)